src/components/token/TokenBalances.js

```diff
@@ -366,6 +379,51 @@ function TokenBalances({ maintenanceMode }) {
       {tokenId && renderTokensTable()}
     </div>
   );
+
+  const renderNewUi = () => (
+    <div className="container-title-page">
+      <p className="title-page">Token Balance</p>
+      {renderSearchField()}
+
+      <div className="token-balances-information-wrapper">
+        <h1>
+          {tokenName === undefined
+            ? `${serverInfo.native_token.name} (${serverInfo.native_token.symbol})`
+            : `${tokenName} (${tokenSymbol})`}
```

Comment:
I think this changed from `name - symbol` to `name (symbol)`.
If this was intended you can disregard this comment.

Reply:
Yes, it was intentional to align with the Figma prototype design.